src/lib/y2users/linux/writer.rb

```diff
@@ -225,6 +228,21 @@ def add_group(group, issues)
         log.error("Error creating group '#{group.name}' - #{e.message}")
       end
 
+      # Applies changes for the login settings
+      def edit_login(_issues)
+        return unless config.login?
+
+        # resetting Autologin settings
+        Yast::Autologin.Disable
+
+        Yast::Autologin.user = config.login.autologin_user&.name.to_s
```

Comment:
how it behave if user is nil or empty? it writes there empty string? Is difference if Use(false) is set and if user is set to empty string?

Reply:
With `Autologin.Use(false)` the method `Autologin.Write` does nothing. Note that Autologin allows to configure both, the *autologin* user and also the *password less login* property. With `Autologin.Use(false)` none of these are written. For disabling autologin, we need to set  `Autologin.Use(true)` and set an empty string as autologin username.